bgmi/config.py

```diff
@@ -295,6 +303,13 @@ def write_config(
 TRANSMISSION_RPC_USERNAME = "your_username"
 TRANSMISSION_RPC_PASSWORD = "your_password"
 
+# qbittorrent-webapi
+QBITTORRENT_HOST = "127.0.0.1"
+QBITTORRENT_PORT = "8082"
+QBITTORRENT_USERNAME = "your_username"
```

Comment:
transmission RPC daemon will omit username and password without auth config, so default `TRANSMISSION_RPC_USERNAME` and `TRANSMISSION_RPC_PASSWORD` could be anything. Will QBittorrent do the same?

Reply:
yes.It omit username and password if enable `Bypass authentication for clients *`